src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+@use '../utils/mixins' as *;
+
+.stars {
+  display: flex;
+  gap: 4px;
+
+  &--1>:nth-child(-n + 1),
```

Comment:
```suggestion
  &--1:nth-child(-n + 1),
```

Reply:
If I do that, the entire background of stars block changes when only background of stars__star elements should change.

Reply:
Eventually `&--1 :nth-child(-n+1)` does work